src/auditor/credentials_template.py

```diff
@@ -1,6 +1,9 @@
 """
```

Comment:
**chore**: this file can be ignored by code coverage to get a more accurate percentage.